src/onegov/org/models/ticket.py

```diff
@@ -663,6 +690,23 @@ def extra_data(self):
     def undecided(self):
         return self.event and self.event.state == 'submitted'
 
+    def prepare_delete_ticket(self):
+        if self.event:
+            self.session.delete(self.event)
```

Comment:
Have we fully thought this through, should past events actually ever be deleted?

Reply:
It would only apply to those which have no future occurrence, else it would not be `deletable`. 

`future_occurrences. (https://github.com/OneGov/onegov-cloud/blob/76cd47f707f42b0f41f0a8d4e369fba985c5a966/src/onegov/org/models/ticket.py#L705C34-L705C34)

But maybe we should reconsider it anyway...

Reply:
Yes, but it still seems weird, since events are public information, tickets are not.

Reply:
Yeah. I think we should not do that. What's also just struck me just now is that `view_event` makes a query for the associated ticket. If it does not exist, it will open a new one !?  This might result in unexpected things happening. (https://github.com/OneGov/onegov-cloud/blob/9af704438ccae09b3a416704b4fcc02d99c19d23/src/onegov/org/views/event.py#L256)

This is all getting a bit convoluted.